python/dgl/distributed/dist_context.py

```diff
@@ -93,11 +93,14 @@ def initialize(ip_config, num_servers=1, num_workers=0,
                 'Please define DGL_NUM_CLIENT to run DistGraph server'
         assert os.environ.get('DGL_CONF_PATH') is not None, \
                 'Please define DGL_CONF_PATH to run DistGraph server'
+        formats = os.environ.get('DGL_GRAPH_FORMAT', 'csc')
+        formats = [f.strip() for f in formats]
```

Comment:
formats will be 'csc,csr,coo'.
Does formats = [f.strip() for f in formats] work ?